Makefile

```diff
@@ -527,10 +527,11 @@ source-dist:
 full-source-dist: light-source-dist.tmp
 	# Get all the dependencies downloaded
 	@$(MAKE) -C deps getall NO_GIT=1
+	@$(MAKE) -C stdlib getall
```

Comment:
I guess this was supposed to include Pkg in the source tarballs? Apparently it didn't work: the 1.0.2 tarball doesn't contain it. That's annoying for building distribution packages without network access.

Reply:
Hmm, I'm 90% sure I tried this, but maybe I didn't do it from a clean repo. We should be able to re-upload the source tarballs with this included, @ararslan? I'll have a look at it.

Reply:
Yes we can re-create and re-upload the source tarballs, it's just a bit annoying since we can't go through the  `full-source-dist` Make target; we'll have to do everything manually.

Reply:
I was talking about light-source-dist, but maybe full-source-dist has the same problem.

Reply:
Should it be included there? I interpreted https://github.com/JuliaLang/julia/blob/master/Makefile#L516 as only including code from this repo, and thus Pkg is just an external dependency that  should only be in the full variant, which it seems to be btw?

Reply:
Well, all other external dependencies are libraries that could be installed separately from Julia, and which generally exist as distribution packages. OTC Pkg cannot be used without Julia. Also, in practical terms, if the plan is to move progressively all stdlib modules to separate repos, it would be quite painful for packagers to have to handle all of them manually (and update versions one by one for each new release).